benchmark/performance_utils.py

```diff
@@ -140,6 +154,7 @@ def set_shapes(self, shape_file_path: Optional[List[Any]] = None):
             ):
                 # Merge shapes using subclass-specific logic
                 additional_shapes = self.set_more_shapes()
+                self.shapes = additional_shapes
```

Comment:
`extend` or `in-place` the core_shapes ?

Reply:
The shapes were set at multiple places and I found it very confusing... So this overwrite was for convenience where I could control the shapes to test in set_more_shapes()

Reply:
Added back default shapes.